<?xml version="1.0" ?>
<launch>

  <!-- Start hmi_demo client -->
  <node name="hmi_demo"
  	pkg="aist_routines" type="hmi_demo.py" output="screen">
    <rosparam file="$(dirname)/../config/aist_config.yaml"/>
    <rosparam file="$(dirname)/../config/hmi_demo.yaml"/>
  </node>

</launch>
